src/libs/ReportUtils.js

```diff
@@ -1785,16 +1785,31 @@ function getTransactionReportName(reportAction) {
     });
 }
 
+/**
+ * Get actor name to display in the message preview
+ *
+ * @param {Object} report
+ * @param {Number} actorID
+ * @param {Boolean} shouldShowWorkspaceName
+ * @param {Boolean} shouldUseShortForm
+ * @returns  {String}
+ */
+
+function getActorNameForPreviewMessage({report, actorID, shouldShowWorkspaceName = false, shouldUseShortForm = false}) {
+    return shouldShowWorkspaceName ? getPolicyName(report) : getDisplayNameForParticipant(actorID, shouldUseShortForm);
+}
+
 /**
  * Get money request message for an IOU report
  *
  * @param {Object} report
  * @param {Object} [reportAction={}] This can be either a report preview action or the IOU action
  * @param {Boolean} [shouldConsiderReceiptBeingScanned=false]
  * @param {Boolean} isPreviewMessageForParentChatReport
+ * @param {Boolean} shouldHideParticipantName
```

Comment:
Looks like it is an optional argument.
```suggestion
 * @param {Boolean} [shouldHideParticipantName]
```